app/src/main/java/org/squidfish/tuplenback/data/GameStatsMapper.kt

```diff
@@ -0,0 +1,29 @@
+package org.squidfish.tuplenback.data
+
+import java.lang.IllegalArgumentException
+import org.squidfish.tuplenback.data.room.GameStatsData
+import org.squidfish.tuplenback.models.GameStatsModel
+
+class GameStatsMapper {
+    companion object {
```

Comment:
You can make `GameStatsMapper` `object`. Also have you though about implementing mappers as extension functions/properties?